pkg/s3select/funcEval.go

```diff
@@ -186,6 +186,41 @@ func (reader *Input) evaluateFuncErr(myVal *sqlparser.FuncExpr) error {
 	return nil
 }
 
+//%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% //
+//%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%     PARSE SELECT FOR JSON STARTS  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% //
+//%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% //
+// evaluateFuncErr is a function that flags errors in nested functions.
```

Comment:
Minio codebase doesn't use this kind of section marking comment style - please change these so they look like the rest of the code.